src/lib/storage/zero_suppression/simd_bp128/simd_bp128_vector.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "storage/zero_suppression/base_zero_suppression_vector.hpp"
+
+#include "oversized_types.hpp"
+#include "simd_bp128_decoder.hpp"
+#include "simd_bp128_iterator.hpp"
+
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * @brief Bit-packed vector with varying bit length
+ *
+ * Values are compressed in blocks of 128 integers. Each block has its own bit-length.
+ * Sixteen blocks combine to form a meta block of 2048 values. Bit-length information
+ * are stored per meta block in 128 bit (8 bit for each bit length).
```

Comment:
I don't like the name "meta block". For me, that would be a block that holds meta information. Instead of "block < meta block", how about:
* "run < block"
* "block < superblock"

To forestall the argument: Just because they might call it meta block in the paper, it does not necessarily mean that we have to do so, too.

Reply:
We don’t have to, but then it would make it even more difficult to understand our code in relation to the paper. I think it’s better to keep the established names.